packages/core/src/components/Spinner/Spinner.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * The `Spinner` component
+ */
+export const Spinner = (props) => {
+  const className = classNames(
+    'ds-c-spinner',
+    props.size && `ds-c-spinner--${props.size}`,
+    props.inverse && 'ds-u-fill--background-inverse ds-u-color--base-inverse',
+    props.background && 'ds-c-spinner--background',
+    props.className
+  );
+
+  return <span className={className} />;
+};
+
+Spinner.propTypes = {
+  /**
+   * Additional classes to be added to the spinner element.
+   * Useful for adding utility classes.
+   */
+  className: PropTypes.string,
+  /** Applies the inverse theme styling */
+  inverse: PropTypes.bool,
+  /** Adds a background behind the spinner for extra contrast */
+  background: PropTypes.bool,
```

Comment:
Oof, good catch. It should be `inversed`, which resembles other boolean HTML attributes like `checked`, `selected`, and `disabled`. I'll open a separate ticket for that. (This naming convention is documented in the [last point here](https://github.com/cmsgov/design-system/wiki/coding-guidelines#react)). 

For this one, we should try making it more clear that it's a boolean prop. `backgrounded` doesn't make sense 🙈 , but I think `hasBackground` would be okay. (Open to other ideas)